model/ftn/ww3_bounc.ftn

```diff
@@ -379,7 +389,14 @@
             END SELECT !GTYPE
           END IF
         END DO
-
+!/RTD
+!/RTD        ! Convert grid boundary cell locations to standard pole
```

Comment:
Is it necessary to include an IF statement to catch the case where /RTD has been set in order for a parent grid to generate BCs for a rotated nested grid, but the parent grid itself is not rotated? In #7 this is achieved using a condition on POLAT

Reply:
Yes - although Carsten's RTDB changes are not in `develop` at the moment (from which this bugfix PR is branched). Technically, handling that scenario should go in the Cartsten's PR. However, it is probably cleaner to update `ww3_bounc` and `ww3_bound` in this PR and cross reference Carsten's PR. I shall do that if all are agreed...

Reply:
Adding that revision in this PR would be OK with me

Reply:
Addressed in latest commit: https://github.com/ukmo-waves/WW3/pull/10/commits/cc0a20474d3941a1a757690d8973fd763e05fb75

Reply:
Nice one, thanks @ukmo-ccbunney